dfsclient/src/main/java/org/hpccsystems/dfs/client/RowServiceOutputStream.java

```diff
@@ -287,16 +374,19 @@ private static class RowServiceResponse
 
     private String makeGetVersionRequest()
     {
-        final String versionMsg = RFCCodes.RFCStreamReadCmd + "{ \"command\" : \"version\", \"handle\": \"-1\", \"format\": \"binary\" }";
+        final String trace = traceContextHeader != null ? "\"_trace/traceparent\" : \"" + traceContextHeader + "\",\n" : "";
```

Comment:
@jakesmith FYI all HPCC4J requests into DAFILESrv will contain a new header with OTel trace context. We've confirmed on our end that the header does not seem to break the server side, but we'd like to get your thoughts. Also, we're opening a jira requesting that dafilesrv at minimum consume this info and propagate it, and optimally add helpful instrumentation based on jtrace.

Reply:
@jpmcmu - is a backslash in an JSON identifier valid? Regardless it looks a little odd, why not just "traceparent" or if want a separate node, generate '"_trace" : { "traceparent": traceContextHeader }' ?

Reply:
@jakesmith it's supposed to be 
```
"_trace": { "traceparent": "00-4bf92f3577b34da6a3ce929d0e0e4736-00f067aa0ba902b7-00" } ,
```
as you point out. Once fixed, will that layout work for your side?

Reply:
Personally, I would make it :
"trace": { "parent": "00-4bf92f3577b34da6a3ce929d0e0e4736-00f067aa0ba902b7-00" }

Reply:
The leading underscore comes from the roxie protocol, I vote we keep it for symetry unless it breaks some sort of convention

Reply:
right ok, and "traceparent" is also part of that protocol?

Reply:
yes, it was recently added: https://hpccsystems.atlassian.net/issues/HPCC-31685

Reply:
ok. So bar the fix to the format, it seems fine.